api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -285,4 +221,86 @@ public ReportingQueryServiceImpl(JdbcTemplate jdbcTemplate) {
                 .userId(rs.getLong("user_id"))
                 .username(rs.getString("username")));
   }
+
+  @Override
+  public List<ReportingWorkspace> getWorkspaces(long limit, long offset) {
```

Comment:
Is the only change here to add limit and offset?

Reply:
yes, though it's  not intended for callers to use this directly.

Reply:
thx